### ISSUE TITLE
VHOST panel does not show up

### ISSUE THREAD

**Issue by tomasliumparas** [allinurl/goaccess#438]
Hi,

I was wondering if I am doing something wrong or.. I can not seem to get vhost panel displayed. 
I am sure it isn't disabled in the config. 

Here is my log format:
`log-format [client %h] [%d %t] [vhost %v] [%bbytes] [STATUS %s] [s/us %^/%D] %m %U %H [ref %R] [agent %u]`

And here is two lines from the log itself:
```
[client x.x.x.x] [2016-06-12 04:31:49] [vhost vhost1.com] [420bytes] [STATUS 200] [s/us 0/478] GET /style.css HTTP/1.1 [ref "http://vhost1.com"] [agent "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.2125.101 Safari/537.36"]
[client x.x.x.x] [2016-06-12 04:35:10] [vhost vhost2.com] [-bytes] [STATUS 302] [s/us 0/56479] GET / HTTP/1.1 [ref "-"] [agent "Mozilla/5.0 (Windows NT 5.1; rv:9.0.1) Gecko/20100101 Firefox/9.0.1"]
```

Any ideas?

**Comment by allinurl** [comment]
For these lines:

    [client 192.168.0.100] [2016-06-12 04:31:49] [vhost vhost1.com] [420bytes] [STATUS 200] [s/us 0/478] GET /style.css HTTP/1.1 [ref "http://vhost1.com"] [agent "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.2125.101 Safari/537.36"]
    [client 192.168.0.100] [2016-06-12 04:35:10] [vhost vhost2.com] [-bytes] [STATUS 302] [s/us 0/56479] GET / HTTP/1.1 [ref "-"] [agent "Mozilla/5.0 (Windows NT 5.1; rv:9.0.1) Gecko/20100101 Firefox/9.0.1"]

this works:

    goaccess -f ../log --log-format='%^ %h] [%d %t] [%^ %v] [%bbytes] [%^ %s] [%^ %^/%D] %m %U %H [%^"%R"] [%^"%a"]' --date-format='%Y-%m-%d' --time-format='%T'

**Comment by allinurl** [comment]
BTW, your format works for me too. Which version are you running?

**Comment by tomasliumparas** [comment]
Resolved, my bad. Just found out that vhosts panel was implemented from version 0.9.5
I am on CentOS, installed version 0.9.3 from epel repo. 

Couple minutes ago installed v.1, everything works like a charm! 
Thanx! :)

**Comment by allinurl** [comment]
Glad it is working now :)

Closing this, feel free to reopen it if needed.
